components/compliance-service/reporting/relaxting/indices_test.go

```diff
@@ -352,3 +353,17 @@ func TestIndexDatesNonLeapYear(t *testing.T) {
 			esIndex)
 	}
 }
+
+func TestIndexDatesSingleFullYearRightUpToFinal59Seconds(t *testing.T) {
+	// Let debug logs show up when running tests
+	logrus.SetLevel(logrus.DebugLevel)
+	startTime := "2018-01-01T00:00:00Z"
+	endTime := "2018-12-31T23:59:59Z"
+	esIndex, err := relaxting.IndexDates(relaxting.CompDailySumIndexPrefix, startTime, endTime)
+
+	if assert.NoError(t, err) {
+		assert.Equal(t, fmt.Sprintf("%[1]s2018.01*,%[1]s2018.02*,%[1]s2018.03*,%[1]s2018.04*,%[1]s2018.05*,"+
```

Comment:
Thank you for the fix Rick!
Should this return just `"%s2018.*"` ?

Reply:
I could have but because of the way that i enter here with this new condition, it returns the year in the form of months.. i can look deepr and get it to return the whole year instead as you mention but functionally, it would make no diff, so i didn't spend any more time

Reply:
Ok, we can leave it like this for now. 👍